### DISCUSSION TITLE
[Bug]Display render mode not work

### DISCUSSION THREAD

**Discussion by Damonvvong** [community/community#22519] [Projects and Issues]
<p>RT, <a href="https://github.com/SwiftOldDriver/iOS-Weekly/blob/197fd25f9b9bfc2252da943af32626a760c2a7c8/Reports/2022/%23190-2022.01.24.md" class="inline-onebox" rel="noopener nofollow ugc">iOS-Weekly/#190-2022.01.24.md at 197fd25f9b9bfc2252da943af32626a760c2a7c8 · SwiftOldDriver/iOS-Weekly · GitHub</a> this project display render mode not work</p>
<p>should i setting some opthions or what should i do?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi <a class="mention" href="/damonvvong">@Damonvvong</a>, I’m not sure what exactly the issue is. You could also consider opening a ticket over at the repo: <a href="https://github.com/SwiftOldDriver/iOS-Weekly/issues" class="inline-onebox" rel="noopener nofollow ugc">Issues · SwiftOldDriver/iOS-Weekly · GitHub</a></p>
